examples/grpc-secured-unary/grpc_secured_unary_service_client.out

```diff
@@ -7,5 +7,5 @@
 # Execute the following command to build the 'client' module.
 $ ballerina build client
 
-# Run the client using the following command.
-$ ballerina run target/bin/client.jar
+# Run the client using the following command and passing Ballerina home path as system property.
```

Comment:
```suggestion
# To run the client, execute the below command by passing Ballerina home path as a system property.
```

Reply:
suggestion: To run the client, execute the below command by passing the path to the Ballerina home directory using the --b7a.home parameter.